app/pdfs/master_skill.rb

```diff
@@ -0,0 +1,44 @@
+class MasterSkill < Prawn::Document
+  def initialize(os)
+    super()
+
+    repeat :all do
+      # header
+      bounding_box [bounds.left, bounds.top], width: bounds.width do
+        text os.title, align: :center, size: 25
+        stroke_horizontal_rule
+      end
+
+      # footer
+      bounding_box [bounds.left, bounds.bottom + 50], width: bounds.width do
+        stroke_horizontal_rule
+        move_down(5)
+        font_size(8) do
+          text "Created for #{os.creator_name}"
+          text "Last updated #{I18n.l(os.updated_at, format: :mdY)}"
+        end
+      end
+    end
+
+    bounding_box([bounds.left, bounds.top - 50], width: bounds.width, height: bounds.height - 100) do
+      text os.organization_title
+      text os.occupation_title
+      text os.industry_title if os.industry
+      move_down 20
+
+      text "Work Processes", size: 16, style: :bold
+      os.occupation_standard_work_processes.each do |oswp|
+        text oswp.work_process.title
+        text "Hours: #{oswp.hours}"
+        move_down 5
+      end
+
+      move_down 20
+      text "Skills", size: 16, style: :bold
+      os.occupation_standard_skills.each do |oss|
+        text oss.skill.description
+        move_down 5
+      end
```

Comment:
Its not important to get this right in this stage, but I'd think of these as nested attributes for the standard. In typical display, you'd show the work process and under that, the skills for that work process.

Thinking about this further bc of the joins, we may need to include the `occupation_standard_work_process_id` on the `OccupationStandardSkill` table to do this correctly. That's not something we have to deal with on workhands to maintain that relationship.

I'd have to look another look at the schema but am thinking it might not make sense to attach a `Skill` to a `WorkProcess` directly because of this.

Reply:
Yes, that would be great to get that straightened out, because I feel like we can end up with weird integrity issues with the `skills.work_process_id` and also the other join tables.

Reply:
Yeah - let's do this: https://github.com/WorkHands/RAPiDSkills/issues/63. It doesnt need to hold up this PR. That relationship is specific to the `occupation_standard`. 

I _think_ that's the last of these modifications with work processes / skills after looking at the schema and thinking about this further. The joins definitely tripped me up here so I overlooked the relationship differences vs workhands. 

Sorry for the rework!